Clarify the anomalocaris mission (#12339)
endless-sky/endless-sky: data/hai/unfettered side missions.txt

Reviewer: Here would be a much natural place to set up the exact terms of the fight.
```suggestion
			`	All of a sudden, she raises her voice for everyone to hear. "I, Mera, challenge you! It will be a one-on-one duel; you in the Anomalocaris against my own champion. The winner will earn the right to own this honorable relic and its mighty weapons!"`
```

			`As you take off to join your fleet in orbit, you receive a hail from the Unfettered. "You must leave your fleet on hold here. If they help you, the duel will be forfeit!"`
				decline
			label anomaly
			`You are hailed by the Unfettered as soon as you reach orbit. "You think us blind, monkey? You are not flying the relic as your flagship! You're not even worth a launch sequence; the duel is forfeit," they say angrily.`
			action
				"reputation: Hai (Unfettered)" = "old reputation: Hai (Unfettered)"
				fail
	on enter "Ehma Ti"
		conversation
			branch anomaly
				not "flagship model: Anomalocaris"
			branch "too many ships"
				"total ships" > 1
			`As you enter the system you can see your challenger is indeed waiting for you, and they promptly send you a hail. "Show us what you've got, monkey! Show us that you control your weapons well, as any warrior should! Prove to us that you have the strength to disable our ships, yet still have the restraint not to destroy them!"`
				accept
			label "too many ships"

> 			`As you arrive with your fleet in this system, you receive a hail from the Unfettered. "If you can't even count how many ships you have, you're not worthy of our time, monkey! Dishonor will get you nowhere with us True Hai! Leave."`

Reviewer 2: Could potentially expand a bit more on this (Leave - we won't fight someone who can't do math!), up to you

Author: well they already speak of not being able to count that might be a little over the top I like the idea though haha

				goto failed
			label anomaly

> 			`You are hailed by the Unfettered as you enter the system. "Monkey! Why do you refuse to use the relic? Only by winning with the relic can you prove yourself worthy of it. Leave - we will not waste anymore of our time with the likes of you."`

Reviewer 2: ```suggestion
			`You are hailed by the Unfettered as you enter the system. "Monkey! Why do you refuse to use the relic? Only by winning with the relic can you prove yourself worthy of it. Leave - we will not waste any more of our time with the likes of you."`
```

			`As you take off to join your fleet in orbit, you receive a hail from the Unfettered. "You must leave your fleet on hold here. If they help you, the duel will be forfeit!"`
				decline
			label anomaly

> 			`You are hailed by the Unfettered as soon as you reach orbit. "You think us blind, monkey? You are not flying the relic as your flagship! You're not even worth a launch sequence, the duel is forfeit," they send in an angry voice.`

Reviewer 3: How do they "send" a message in an angry voice? Also, comma splice, you could also use a period instead of a semicolon.

```suggestion
			`You are hailed by the Unfettered as soon as you reach orbit. "You think us blind, monkey? You are not flying the relic as your flagship! You're not even worth a launch sequence; the duel is forfeit," they say angrily.`
```

> 			`	"You also need to only have one ship, although we can tolerate leaving your fleet on hold in this system."`

Reviewer 3: I think this is better?

```suggestion
			`	"You also need to only have one ship, although we can tolerate you leaving your fleet on hold in this system."`
```

				to display

> 					"total ships" > 1

Reviewer 4: This seems confusing. She just said the thing is forfeit, but now she's saying they will tolerate leaving my extra ships in this system? Did I fail or do I just need to have the other ships hold position and  then go to the next system?
We can probably just remove this. If a person got here, they need to reload to actually do the mission anyway, and if they then take off with multiple ships, they'll get the reminder to have the others hold position.
```suggestion
```

Author: the mission only fails when you enter the other system with a fleet, it's fine to leave them on hold in this system

Reviewer 4: The very next line is an `action` that fails this mission. Players seeing this branch have failed because this is the branch for not having the correct flagship.

Author: true sorry I assumed this was at another place, thanks :)

			action
				"reputation: Hai (Unfettered)" = "old reputation: Hai (Unfettered)"
				fail
	on enter "Ehma Ti"
		conversation
			branch anomaly
				not "flagship model: Anomalocaris"
			branch "too many ships"
				"total ships" > 1
			`As you enter the system you can see your challenger is indeed waiting for you, and they promptly send you a hail. "Show us what you've got, monkey! Show us that you control your weapons well, as any warrior should! Prove to us that you have the strength to disable our ships, yet still have the restraint not to destroy them!"`
				accept
			label "too many ships"
			`As you arrive with your fleet in this system, you receive a hail from the Unfettered. "If you can't even count how many ships you have, you're not worthy of our time, monkey! Dishonor will get you nowhere with us True Hai! Leave."`
				goto failed
			label anomaly

> 			`You are hailed by the Unfettered as you enter the system. "Monkey! Why do you refuse to use the relic? Only by winning with the relic can you prove yourself worthy of it. Leave, we will not waste anymore of our time with the likes of you."`

Reviewer 3: Another comma splice

```suggestion
			`You are hailed by the Unfettered as you enter the system. "Monkey! Why do you refuse to use the relic? Only by winning with the relic can you prove yourself worthy of it. Leave - we will not waste anymore of our time with the likes of you."`
```

			dialog `You receive a hail from Mera, who sounds quite angry. "Destroying the ship can only be due to either incompetence or evil intent, and we will not tolerate either!"`
			"reputation: Hai (Unfettered)" = "old reputation: Hai (Unfettered)"
			fail
		conversation
			branch cheat
				"total ships" > 1
			`You receive a message from the Unfettered ship. "We surrender. Land on Firelode to receive your reward."`
				accept
			label cheat

> 			`You come here with a fleet and have the audacity to disable my ship? Your dishonor knows no bound, monkey!`

Reviewer 3: Who's saying this?

(Same question for the other place it appears).

Author: the captain of the ship when you disable him (with a fleet or not)

Reviewer 3: Ah, in that case it just needs quotes.

> 			`You come here with a fleet and have the audacity to disable my ship? Your dishonor knows no bound, monkey!`

Reviewer 4: ```suggestion
			`"You come here with a fleet and have the audacity to disable my ship? Your dishonor knows no bound, monkey!"`
```

			dialog `You receive a hail from Mera, who sounds quite angry. "Destroying the ship can only be due to either incompetence or evil intent, and we will not tolerate either!"`
			"reputation: Hai (Unfettered)" = "old reputation: Hai (Unfettered)"
			fail
		conversation
			branch cheat
				"total ships" > 1
			`You have managed to defeat the Unfettered challenger. Now it's time to land on Firelode and claim your reward.`
				accept
			label cheat

> 			`You come here with a fleet and have the audacity to disable my ship? Your dishonor knows no bound, monkey!`

Author: ```suggestion
			`"You come here with a fleet and have the audacity to disable my ship? Your dishonor knows no bound, monkey!"`
```
